src/pages/dongmin/Main/Feeds/Components/Comment.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './Comment.scss';
+
+const Comment = props => {
+  const { index, name, content, deleteCommentFunc } = props;
+
+  const isHeartEmpty = src => (src.indexOf('empty') !== -1 ? true : false);
```

Comment:
indexOf는 몇 번째에 인자로 넘겨준 문자열이 있냐를 판단합니다. 없다면 -1을 반환하게 되는데 `String.includes`를 쓰면 우리가 원하는 boolean값을 바로 반환 받을 수 있기 때문에 이렇게 바꾸는게 조금 더 좋을 것 같습니다.
```suggestion
  const isHeartEmpty = src => src.includes('empty');
```

하지만 like를 관리하는 state가 있다면 더 구현하기 쉬울 것 같습니다.

Reply:
like를 관리하는 state는 Comment.js에 있어야 할까요? 상위에 있어야 할까요?

Reply:
includes 메서드가 익숙하지 않아 생각을 못했네요!
직관적으로 생각했을 때는 댓글을 각각 관리해야되기 때문에 state도 Comment 컴포넌트 내에 있어야 할 것 같습니다. 피드백 반영하면서 더 생각해보겠습니다!

Reply:
```js
  const [like, setLike] = useState('/images/dongmin/empty-heart.png');

  const onBtnClickedLike = () => {
    like.includes('empty')
      ? setLike('/images/dongmin/full-heart.png')
      : setLike('/images/dongmin/empty-heart.png');
  };
```
```jsx
<button className="like" onClick={onBtnClickedLike}>
  <img alt="like" src={like} />
</button>
```
Comment 컴포넌트 내에 state 세워서 구현했습니다. 더 나은 방법이 있을까요?!

Reply:
Comment 컴포넌트내에서 관리하면 CommentList와 Comment가 엮이려면 또 하나의 코드가 필요하고, 불필요한 로직이 추가 될 것 같습니다. 
like는 CommentList에서 각각의 item을 객체를 통해 관리하는게 더 좋을 것 같습니다.

Reply:
피드백 내용을 이해하지 못했습니다 ㅜㅜ 댓글 좋아요 버튼이 Comment 컴포넌트 내에 있는 상황에서 컴포넌트 내에서 처리하는 게 더 코드가 간결하지 않을까 라는 생각이 들어서요. 혹시 피드에 종속된 좋아요 버튼 까지 같이 관리할 경우를 포함해서 말씀하신 걸까요?

Reply:
라운지 타임 때 설명해주셔서 이해했습니다!